ibi-manifest.template

```diff
@@ -1,2 +1,24 @@
-{"version":"4.14.5","domain":"${DOMAIN}","cluster_name":"${CLUSTER_NAME}","cluster_id":"${CLUSTER_ID}","master_ip":"${MASTER_IP}","release_registry":"quay.io","hostname":"${HOSTNAME}"}
-
+{
+  "api_version": 1,
+  "base_domain": "${DOMAIN}",
+  "cluster_name": "${CLUSTER_NAME}",
+  "cluster_id": "${CLUSTER_ID}",
+  "node_ip": "${NODE_IP}",
+  "release_registry": "quay.io",
+  "hostname": "${HOSTNAME}",
+  "KubeconfigCryptoRetention": {
+    "KubeAPICrypto": {
```

Comment:
Didn't realize the capitalization is messed up like this. We'll have to fix that